3-extra/1-guess-the-number-project/script.js

```diff
@@ -1,23 +1,52 @@
 let randomNumber = Math.floor(Math.random() * 100 + 1);
+let display = document.querySelector(".final-output");
+let tries = document.querySelector(".Tries-output");
+let countTries = 7;
 
-function guessNumber() {
   //Collect input from the user
-  let guess = document.querySelector(".inputs-Values").value;
-
   //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
-
   //If the users guess is higher than the random number print Number is too high, try again (hint use .final-out class to print)
-
   //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
-
   //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
-}
+
+  function guessNumber() {
+
+    let guess = document.querySelector(".inputs-Values").value;
```

Comment:
What is the type of `guess`?

Reply:
I didn't think about it, but it's interesting because <input type="number"> but typeof guess is string.

Reply:
Yeah, the value of an element is always a string as far as I can tell, even if the `type` attribute is set to "number".  So you always need to pass it through `parseInt(input, 10)` and check for `NaN`